### PR TITLE
Only fail empty function blocks if there's a corresponding call block

### PR DESCRIPTION
**Pull request by joshlory**
Sample level: http://localhost-studio.code.org:3000/s/course3/stage/5/puzzle/6

### CODE REVIEW

**Comment by joshlory** [comment]
[Only fail on empty function blocks if a corresponding function call block is present](https://trello.com/c/ZGmMK0pa/97-only-fail-on-empty-function-blocks-if-a-corresponding-function-call-block-is-present)

File: apps/src/feedback.js [hunk 1]
```diff
@@ -1266,14 +1266,22 @@ FeedbackUtils.prototype.getEmptyContainerBlock_ = function () {
...
   if (!emptyBlock) {
     return TestResults.ALL_PASS;
   }
 
-  var type = emptyBlock.type;
+  const type = emptyBlock.type;
   if (type === 'procedures_defnoreturn' || type === 'procedures_defreturn') {
-    return TestResults.EMPTY_FUNCTION_BLOCK_FAIL;
+    const findUsages = block =>
+      block.type === type.replace('_def', '_call') &&
```
Comment (balderdash) [inline apps/src/feedback.js]: Instead of `type.replace(...)`, you should be able to use `emptyBlock.getProcedureInfo().callType`.

Comment (joshlory) [reply apps/src/feedback.js]: Good call, updated!

**Comment by joshlory** [comment]
+cc @kiki-lee

**Pull request merged**